plugin/storage/integration/elasticsearch_test.go

```diff
@@ -249,3 +254,28 @@ func (s *ESStorageIntegration) cleanESIndexTemplates(t *testing.T, prefix string
 	}
 	return nil
 }
+
+func (s *ESStorageIntegration) testArchiveTrace(t *testing.T) {
```

Comment:
what's special about this test compared to the regular write-tests?

Reply:
It adds a test that is before the configured max span age to test that it can be retrieved from archive storage. It looks like it was only catered to ES storage.

Reply:
I suggest documenting it in the code, to call attention to what's in the test that is unusual. Should the start time be computed relative to maxspanage?

Reply:
@yurishkuro Done!